apps/src/templates/progress/progressStyles.js

```diff
@@ -232,30 +232,9 @@ function shapeSizeStyle(shape, size) {
  */
 
 /**
- * Get border and background styling based on level and student progress.
- *
- * Note: `levelProgressStyle` below is used by our legacy bubble components,
- * but once those get wrapped into our new components these two functions can
- * be merged into one.
- */
-export function getProgressStyle({
-  levelStatus,
-  levelKind,
-  isDisabled,
-  isBonus
-}) {
-  return {
-    ...(!isDisabled && hoverStyle),
-    ...levelProgressStyle(levelStatus, levelKind, isDisabled),
-    ...(isDisabled && isBonus && bubbleStyles.bonusDisabled)
-  };
-}
-
-/**
- * Given a level object, figure out styling related to its color, border color,
- * and background color
+ * Get border and background styling based on level kind and student progress.
  */
-export const levelProgressStyle = (levelStatus, levelKind, disabled) => {
+export function levelProgressStyle(levelStatus, levelKind) {
```

Comment:
I think the intent here is that levelStatus and/or levelKind can be be undefined here, right?  If so, could you add that information to the header comment and add tests for those cases?

Reply:
turns out we don't have any tests for this module yet. would you be okay with me deferring that work until the PR that actually consolidates our use of this stuff through `BubbleFactory` (or whatever that evolves into)?